config.yml.example

```diff
@@ -26,7 +26,7 @@ auth_file: "/home/git/.ssh/authorized_keys"
 
 # Redis settings used for pushing commit notices to gitlab
 redis:
-  bin: /usr/bin/redis-cli
+  bin: /home/git/gitlab-shell/bin/dummy-redis-cli
```

Comment:
If I remember right, it will cause the post-receive hook to fail at the point where it tries to call redis.
The post-receive part we need (i.e. the rabbitmq part) is after the redis part, so it would prevent the Core from learning about git repo updates.